Library/Homebrew/formula.rb

```diff
@@ -730,12 +735,32 @@ def bottle_prefix
     prefix+".bottle"
   end
 
-  # The directory where the formula's installation logs will be written.
+  # The directory where the formula's installation or test logs will be written.
   # @private
   def logs
     HOMEBREW_LOGS+name
   end
 
+  # The prefix, if any, to use in filenames for logging current activity
```

Comment:
It would be nice if the documentation comments were always complete sentences, i.e. I'm missing a trailing full stop here and in a few other places that is present in (most?) other such comments.